web/src/app/projects/view/view.component.ts

```diff
@@ -123,6 +151,17 @@ export class ViewProjectComponent implements OnInit, OnDestroy {
     return this.project.isAdmin(this.authService.profile.uid);
   }
 
+  /**
+   * Set the followers flag true if user uid is present in the database else set false
+   */
+  public setFollower(): void {
```

Comment:
I think `isFollower()` would make more sense, then this can be called where needed